Tests/MongoKittenTests/ConnectionSettingsTests.swift

```diff
@@ -36,4 +36,52 @@ class ConnectionSettingsTests : XCTestCase {
             }
         }
     }
+    
+    func test_findAndModify() throws {
```

Comment:
This can probably be removed, as it's really an example of your problem with transactions.

Reply:
I’ll get these removed in a cleanup PR here soon.